benchmark/test/reference/preconditioner.precond.stderr

```diff
@@ -1,7 +1,7 @@
 Running on ReferenceExecutor
 Running with 2 warm iterations and 10 running iterations
 The random seed for right hand sides is 42
-Running with preconditioners: jacobi
-Running test case stencil(100, 7pt)
+
+Running test case 
+    {"operator":{"stencil":{"name":"7pt","size":100}},"preconditioner":{"type":"preconditioner::Jacobi","max_block_size":32,"storage_optimization":[0,0],"skip_sorting":true},"format":"csr"}
```

Comment:
That's again one of these places where I think giving names to inputs and benchmarks would be useful